receiver/sqlqueryreceiver/logs_receiver.go

```diff
@@ -316,16 +316,24 @@ func (queryReceiver *logsQueryReceiver) storeTrackingValue(ctx context.Context,
 }
 
 func rowToLog(row sqlquery.StringMap, config sqlquery.LogsCfg, logRecord plog.LogRecord) error {
-	logRecord.Body().SetStr(row[config.BodyColumn])
+	var errs []error
+	var bodyColumnName = config.BodyColumn
```

Comment:
I don't see any extra benefit to using a temporary variable here so I'd prefer to just use `config.BodyColumn`, unless I'm missing something.

Reply:
Temporary variable is also used in line [323](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35189/files#diff-0c9a67fda52566bc14742cf772b15a23ac8c74c66e5fc4f9b86c734b809d4a09R323), in the error.

Please confirm if you prefer two calls to `config.BodyColumn`, or if I should leave temporary variable.

Reply:
Yes, I'd prefer to just use `config.BodyColumn` and remove the temporary variable 👍

Reply:
Temporary variable removed in [69e8f54](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35189/commits/69e8f54276f79d3081796de6c3269d4d73f6d8e6)